modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/dashboard/ProductPerfomance/ProductPerfomanceTypes.ts

```diff
@@ -12,51 +12,82 @@
  * details.
  */
 
-export type DataProperties = {
+export type DataPropertiesTypes = {
```

Comment:
The path name productPerformance is wrong, please change it. The name of this file it is wrong too.